assets/schema_tools.json

```
@@ -23,7 +23,22 @@
                 "type": "string",
                 "meta": ["aligner"],
                 "pattern": "^\\S+$",
-                "errorMessage": "align name must be provided and cannot contain spaces",
+                "enum": [
+                    "CLUSTALO",
+                    "FAMSA",
+                    "KALIGN",
+                    "LEARNMSA",
+                    "MAFFT",
+                    "MAGUS",
+                    "MUSCLE5",
+                    "TCOFFEE",
+                    "REGRESSIVE",
+                    "UPP",
+                    "3DCOFFEE",
+                    "MTMALIGN",
+                    "FOLDMASON"
+                ],
+                "errorMessage": "align name must be provided, has to be a string, cannot contain spaces and must be one of: CLUSTALO, FAMSA, KALIGN, LEARNMSA, MAFFT, MAGUS, MUSCLES, TCOFFEE, REGRESSIVE, UPP, 3DCOFFE, MTMALIGN or FOLDMASON",
```

I am not sure we want to make this fail. It's a nice addition, but providing a wrong value won't make the pipeline fail, it will just not run. 
In any case if we add this here we should update the docs on how to contribute.
What is your opinion @luisas

---

I think it is fine that we let it fail, but yes we need to update the contributing logs! :) 

We should add this to this PR before merging - I will try to do it today :)

---

Awesome thank you! Let me know if you want me to do anything else :)

---

@luisas any news on this? :)

---

@nvnieuwk upsi in the end I forgot, was busy with the other hackathon project! Thanks for the reminder :) 

I added one line in the docs, i believe for now that is fine!

---

I will merge once the tests pass :)

---

Awesome, thank you!